### DISCUSSION TITLE
Database querying

### DISCUSSION THREAD

**Discussion by lvothnrv** [strapi/strapi#8029] [Legacy Topics]
Hi friends, I would need your help please.
I would like to know how I could only display 'Produits' that have a 'Stock' greater than 0.
<img width="969" alt="Capture d’écran 2020-09-24 à 17 52 53" src="https://user-images.githubusercontent.com/62966130/94154723-3fa86380-fe8f-11ea-8d93-35533c797f28.png">

**Comment by derrickmehaffy** [comment]
Is this for the Admin or from the user APIs? (REST & GraphQL)

**Reply by lvothnrv** [reply]
This is for user api.

**Reply by derrickmehaffy** [reply]
`localhost:1337/produits?stock_gt=0` You can find the filters and query parameters in the documentation here :wink: 
https://strapi.io/documentation/v3.x/content-api/parameters.html

**Reply by lvothnrv** [reply]
I have this error :( in fact it doesn't work because my Stock is in a repeatable component and I don't know how to do this.

{"statusCode":400,"error":"Bad Request","message":"Your filters contain a field 'Stock' that doesn't appear on your model definition nor it's relations"}

**Reply by derrickmehaffy** [reply]
Yeah components cannot be filtered currently, see: https://github.com/strapi/strapi/issues/5124

**Reply by lvothnrv** [reply]
Oh okok thank's
